wagtail/search/backends/elasticsearch5.py

```diff
@@ -1136,7 +1135,21 @@ def __init__(self, params):
         # Any remaining params are passed into the Elasticsearch constructor
         options = params.pop("OPTIONS", {})
 
-        self.es = Elasticsearch(hosts=self.hosts, timeout=self.timeout, **options)
+        # If HOSTS is not set, convert URLS setting to HOSTS
+        if self.hosts is None:
+            es_urls = params.pop("URLS", ["http://localhost:9200"])
+            # if es_urls is not a list, convert it to a list
+            if isinstance(es_urls, str):
+                es_urls = [es_urls]
+
+            parsed_urls = [urlparse(url) for url in es_urls]
+
+            self.hosts = [self._get_host_config_from_url(url) for url in parsed_urls]
+            options.update(self._get_options_from_host_urls(parsed_urls))
```

Comment:
Just wanted to flag that this means the URLs can override options defined in `OPTIONS`. Is that correct, or should it be the other way around?

I suppose this is necessary if the auth credentials in the URLs differ, but that's no longer supported in ES8.

It doesn't really matter though, as we don't process anything else other than the auth credentials.

Reply:
Yep, I can't really imagine any other scenarios besides the basic auth credentials where we'll need information from the URLS list to be propagated back to the global options, so this will only be an issue if someone has specified a `basic_auth` option and also set conflicting auth details within the URL - and I think it's fair to chalk that up as "undefined behaviour" :-)

Reply:
Sounds good to me!